### ISSUE TITLE
Deployment is broken?

### ISSUE THREAD

**Issue by octogonz** [microsoft/json-schemas#116]
## The problem

We recently merged an update in PR https://github.com/microsoft/json-schemas/pull/115, which can be seen here:

`main` branch:
https://github.com/microsoft/json-schemas/blob/d1890b4b74daa5fc33e029e6c25525e819c4f568/rush/v5/command-line.schema.json#L615

`live` branch:
https://github.com/microsoft/json-schemas/blob/e8f62f335947832dd392c8e6f19e2f71363aa32d/rush/v5/command-line.schema.json#L615

However 24 hours later, the CDN deployed file is still missing the new `choiceList` item:
https://developer.microsoft.com/json-schemas/rush/v5/command-line.schema.json

## Possible regression?

@DevArvindsa Maybe the `json-schemas` deployment was broken by your recent change in https://github.com/microsoft/json-schemas/commit/04df5431a9aaca9aa5ba24d49da36ad23c5f3c3f ?  Could someone from Microsoft investigate? Thanks!

CC @iclanton @apostolisms @EscapeB

**Comment by EscapeB** [comment]
Is there any update? Seems JSON schema still not updated yet...

**Comment by KebapBoy** [comment]
Unfortunately the schemas are still out of date

**Comment by moegelin** [comment]
they are still out of date...

**Comment by octogonz** [comment]
Our plan is to relocate the Rush schemas to the `rushstack.io` server. Let me see if I can prioritize that work.

**Comment by ElizabethSamuel-MSFT** [comment]
@octogonz Can you check if the changes are reflecting now? Thx.

**Comment by octogonz** [comment]
> Can you check if the changes are reflecting now? Thx.

@ElizabethSamuel-MSFT No, if you open https://developer.microsoft.com/json-schemas/rush/v5/command-line.schema.json in a private browser tab and search for `choiceList`, it is still missing in this document. 

If a fix has been made to the deployment pipeline, let me try committing a trivial change to force a redeployment, and see if that fixes it.

**Comment by octogonz** [comment]
After merging PR #126 and waiting 5 minutes, the CDN is still serving the old version of the file.

@ElizabethSamuel-MSFT @DevArvindsa  Just to confirm: The procedure for deploying is as follows:

1. Merge your PR into `main`
2. `git fetch`
3. Checkout `origin/live`
4. Merge `origin/main` into `live`
5. Push to `origin/live`

Is that still the right steps?

**Comment by ElizabethSamuel-MSFT** [comment]
@octogonz: I've confirmed that @DevArvindsa and crew are aware of this issue. I searched for "choiceList" and I see it now. Can you confirm? Thanks.

**Comment by octogonz** [comment]
Yes, it was deployed successfully. 👍

Before closing this issue, to confirm that the pipeline is working, I will make another PR to revert my test change and check that it gets deployed also.

**Comment by octogonz** [comment]
I created PR #127 which reverts PR #126, and merged that into `main` and then into `live`.

@ElizabethSamuel-MSFT @DevArvindsa 🤔 No after merging to `live`, 25 minutes later this URL does not reflect the change:

https://developer.microsoft.com/json-schemas/rush/v5/command-line.schema.json
